optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/ConstraintCollectors.java

```diff
@@ -69,6 +69,21 @@
  */
 public final class ConstraintCollectors {
 
+    public static <A> DefaultUniConstraintCollector<A, ?, Object> loadBalanceByCount() {
```

Comment:
Great that you've implemented it with a ConstraintCollector for loadBalancing.
However, given the complexity of loadBalancing and fairness - this is just the tip of the iceberg, see [1] - I am not comfortable adding it a method name into the backwards compatible API.
Let's have it ripe and mature in the example first. Can you put this code in the tennis example for now? 

[1] https://www.optaplanner.org/blog/2017/02/03/FormulaForMeasuringUnfairness.html

Reply:
Note that even [1] doen't take into account the weird affinity-effects of the task assignment example.

Reply:
+1.